src/model/reference/reference.utils.ts

```diff
@@ -0,0 +1,55 @@
+import { EditorState } from "prosemirror-state";
+import {DOMParser as ProseMirrorDOMParser} from 'prosemirror-model';
+import { getTextContentFromPath, makeSchemaFromConfig } from "../utils";
+import { ReferenceContributor } from "./types";
+import * as referenceInfoConfig from '../config/reference-info.config';
+import xmldom from "xmldom";
+import { JSONObject } from "../types";
+
+const referenceInfoSchema = makeSchemaFromConfig(
+  referenceInfoConfig.topNode,
+  referenceInfoConfig.nodes,
+  referenceInfoConfig.marks
+);
+
+export function createReferencePersonList(referenceXml: Element, groupType: string): ReferenceContributor[] {
+  const contributors = referenceXml.querySelector(`person-group[person-group-type=${groupType}]`);
```

Comment:
This line does return an `Element` but for some reason its `children` property is always undefined when using xmldom.